test/functional/terminal/tui_spec.lua

```diff
@@ -831,6 +831,26 @@ describe('TUI', function()
       {3:-- TERMINAL --}                                    |
     ]=])
   end)
+
+  it('forwards unknown DCS/OSC sequences to outer terminal', function()
```

Comment:
Is it worth testing that with `$TMUX` set, we get the tmux-wrapped escape code?

Reply:
Yup, I guess we could simply faking it by using `let $TMUX = 'nonempty'`. After we get the tests to work at all. :)